ReactiveCocoa/Swift/ObjectiveCBridging.swift

```diff
@@ -15,7 +15,7 @@ extension RACScheduler: DateSchedulerType {
 	}
 
 	public func schedule(action: () -> ()) -> Disposable? {
-		return self.schedule(action)
+		return (self.schedule(action) as RACDisposable) as Disposable?
```

Comment:
Ooooh yes, we were talking about different things.
I meant that in this example:

```swift
protocol Disposable {}
class RACDisposable: Disposable {}

class RACScheduler: DateSchedulerType {
  func schedule() -> Disposable! {
     // This coercion works.
     // But I think it's good to implement the method
     // explicitly for documentation purposes.
     // And I'm not 100% sure Swift would understand that
     // that this DOES implement DateSchedulerType without it.
     return RACDisposable()
  }
}
```